bokeh/_templates/doc_js.js

```diff
@@ -12,7 +12,14 @@ Bokeh.Collections.register_models({
   var websocket_url = null;
 {%- endif %}
 
+{% if comms_target -%}
+  var comms_target = "{{ comms_target }}";
+{%- else %}
```

Comment:
Might be nice to call this notebook_comms_target or somehow be clear it's for the notebook case.